components/clientComponents/forms/NextButton/NextButton.tsx

```diff
@@ -25,7 +25,7 @@ export const NextButton = ({
     return false;
   };
 
-  if (!currentGroup || !hasNextAction(currentGroup)) {
+  if (!currentGroup || currentGroup === "review" || !hasNextAction(currentGroup)) {
```

Comment:
For good measure we likely want to add "end" to the or checks as well --- should be unreachable but just in case